app/views/books/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= @book.title %></h1>
+<h3>Average Rating: <%= @book.avg_rating %>/5</h3>
+<h3>Highest Rating: <%= @book.max_rating %>/5</h3>
+  <p><%= @book.max_rating %></p>
+  <p><%= @book.review_with_max_rating %></p>
```

Comment:
Am I missing something? Believe this returns a `review` object. Would need to call something along the lines of `review_with_max_rating.user.name` (which we would want to push back to the model to keep some of that logic out of the view), and `review_with_max_rating.body`.